invenio_cli/helpers/commands.py

```diff
@@ -151,8 +151,25 @@ def _ensure_containers_running(self):
         docker_helper.start_containers()
         # TODO: Find faster way to procede when containers are ready
         time.sleep(30)  # Give time to the containers to start properly
+    
+    def destroy(self):
+        """Destroys env, containers."""
+        command = [
+            'pipenv', '--rm'
+            ]
+        subprocess.run(command, check=True)
+        click.secho('Destroyed Venv...', fg='green')
+
+        docker_helper = DockerHelper(
+            self.cli_config.get_project_shortname(),
+            local=True)
+        """Stop containers"""
```

Comment:
I think both this and the below (destroy...) comments are not needed. Firstly I would say that function name is quite explicit. However, if you actually want to put the comment use `#`, `"""` is used for docstrings (top of the function) or multiline comments.
```suggestion
```

(Same for `"""Destroy containers"""`)